test/core/linter_test.py

```diff
@@ -371,27 +372,52 @@ def test__linter__encoding(fname, config_encoding, lexerror):
     assert lexerror == (SQLLexError in [type(v) for v in result.get_violations()])
 
 
+# noqa tests require a rule_set, therefore we construct dummy rule set for glob matching.
+dummy_rule_set = [BaseRule(f"L{i:03d}", "dummy_rule") for i in range(1, 100)]
```

Comment:
Will this require updating if/when the number of rules in SQLFluff exceeds 100? Should we try and avoid that, e.g. by building this dummy set based on the actual set of rules?